index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Aamir Asaram</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <img src="assets/images/menu.png" class="menu-btn"">
+    </nav>
+  </header>
+  <div class="bg">
+    <section>
+      <h2 class="blue heading">Hey there. I'm Aamir.</h2>
+      <h1 class="green heading">I'm a software developer</h1>
+      <p class="blue intro">I can help you build a product, feature or website.
+      Look through some of my work and experience! If you like
+      what you see and have a project you need coded, don't hesitate
+      to contact me.</p>
+    </section>
+
+    <div class="social">
+      <ul>
+        <!-- </a> -->
+        <a href="https://twitter.com/aaaaamir_"><li><button class="social-btn"><i class="fa fa-2x fa-twitter"></i></button></li></a>
+        <a href="https://www.linkedin.com/in/aamir-asaram-089802213/"><li><button class="social-btn"><i class="fa fa-2x fa-linkedin"></i></button></li></a>
+        <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-2x fa-medium"></i></button></li></a>
+        <a href="https://github.com/aamir-asaram"><li><button class="social-btn"><i class="fa fa-2x fa-github"></i></button></li></a>
+        <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-2x fa-angellist"></i></button></li></a>
```

Comment:
- [ ] [OPTIONAL] for your future reference I've made this short video below on how to download the icons from Figma
![firefox_o1brwEllrR](https://user-images.githubusercontent.com/104968236/234847862-3e3e677f-5d21-49e6-b6ca-37fbf45ec9d3.gif)